### DISCUSSION TITLE
Are Helix-like unimpaired mode bindings (using tree-sitter) possible?

### DISCUSSION THREAD

**Discussion by meepleek** [71/dance#279] [Ideas]
Helix has some super useful looking bindings like goto_next_function and other similar ones that leverage tree-sitter. 
https://docs.helix-editor.com/keymap.html#unimpaired

Would smt. like that be achievable with Code/Dance?

**Comment by tshort** [comment]
Looks like you can do that in https://github.com/DnailZ/oomotion_vscode. Some interesting ideas there. Wish it were based on Dance.
